Core/Utilities/test/Test_File.py

```diff
@@ -131,10 +132,13 @@ def testGetCommonPath( self ):
 
     # this folder and its parent
     filesList = [ os.path.abspath("..") + os.sep + x for x in os.listdir("..") ] + self.filesList
+    #FIXME: the following lines are here temporary for debugging
+    print 'self.fileList', self.filesList
+    print 'fileList', filesList
     self.assertEqual( getCommonPath( filesList ), os.path.abspath("..") )
```

Comment:
If I put that it fails when I run locally. The test itself is correct, I think there's a protection in Jenkins on listing a directory one level up. I am either removing or changing it.